content/news/release-20180923.md

```diff
@@ -0,0 +1,59 @@
+</div>
+
+Connecting to your [OpenRA forum](https://forum.openra.net) account allows you to verify your identity to other players without revealing your IP address. You can also choose to display up to five badges, with a selection of default options and [custom awards](https://forum.openra.net/viewtopic.php?f=82&t=20613) available to choose from. Server hosts can configure private clan or tournament servers based on accounts instead of using passwords.
+
+Other improvements to the in-game UI include a new preset- and custom-color tab on the color chooser, the ability to select in text-fields using the mouse, and returning you to the appropriate sub-menu (instead of the main menu) after leaving a game.
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20180923-colorpicker.png" width="600" alt="Custom color picker">
+
+The color chooser now allows you to save and restore your favorite colors.
+</div>
+
+The new AppImage package format allows us to provide "upstream" releases that should work on all modern Linux distributions. These self-contained packages do not require installation, and so you can keep multiple versions on your system in order to join servers or watch older replays. AppImages can be integrated into your system (adding launcher shortcuts and version updates) by installing the [appimaged](https://github.com/AppImage/appimaged) daemon. If you prefer traditional packaging systems, we encourage you to try out community-supported packages for your distro.
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20180923-techtooltips.png" width="600" alt="Tech structure tooltips">
+
+Mouse-over a capturable tech structure for a description of what it provides.
+</div>
+
+Notable gameplay changes in this release include:
+
+* Significant improvements to rendering performance and unit responsiveness
+* Descriptive tooltips for tech structures
+* Tech structures and walls remain on the battlefield if their owner is defeated
+* Overhauled Chronoshift-return behaviour in Red Alert
+* Disabled Kill Bounties in the Red Alert campaign and default multiplayer configuration
+* Fixes for several crashes and gameplay bugs in release-20180307
+* A collection of community balance changes in [Red Alert](https://github.com/OpenRA/OpenRA/wiki/Changelog/03a285750bd37bd4a27edcd2e49f43f72eabfbe1#ra-balance-changes) and [Tiberian Dawn](https://github.com/OpenRA/OpenRA/wiki/Changelog/03a285750bd37bd4a27edcd2e49f43f72eabfbe1#cnc-balance-changes)
+
+As always, we suggest you read the [changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/03a285750bd37bd4a27edcd2e49f43f72eabfbe1) for more complete information, and head over to the [download](/download/) page to get started with the new release!
+
+<hr />
+
+The latest [Mod SDK](https://github.com/OpenRA/OpenRAModSDK/releases/tag/20180825) releases contain major improvements over the 20180307 release:
```

Comment:
I guess this should have been updated to 29180923?

Reply:
Good catch! #402